### DISCUSSION TITLE
Drag & drop message

### DISCUSSION THREAD

**Discussion by ch-westphal** [OpenBoard-org/OpenBoard#1182] [Q&A]
Hello,
I'm trying to write an application for OpenBoard (electrical diagrams) that uses drag and drop within the application.
Outside OpenBoard (chrome or FF) it works without a problem but if I try to transform the code into an OpenBoard application, the drag and drop operations result in the message "Are you trying to drop the object(s) inside the widget?" (« Essayez-vous de déposer l’objet à l’intérieur du widget ? » en français) 

Is there a work around this?

Thank you
Christian

**Comment by letsfindaway** [comment]
I think this actually is a problem with OpenBoard, especially as it does not help to answer the message box with "yes". And no, I cannot see a workaround at the moment. You have the same problem when you embed a page like https://www.w3schools.com/HTML/html5_draganddrop.asp, which demonstrates the usage of HTML5 DnD. So I'm creating a bug report from this discussion.
